modules/installation-aws-about-government-region.adoc

```diff
@@ -42,6 +42,14 @@ ifdef::aws-secret[]
 The following AWS Top Secret Region partition is supported:
 
 * `us-iso-east-1`
+
+endif::aws-secret[]
+
+ifdef::aws-secret[]
```

Comment:
You have nested an `ifdef::aws-secret[]` inside another instance of `ifdef::aws-secret` This is not necessary. You can include your MTU statement in the first condition, albeit I suggest your statement be added as a note. Your markdown should look like this:

ifdef::aws-secret[]
The following AWS Top Secret Region partition is supported:

* `us-iso-east-1`
endif::aws-secret[]

[NOTE]
====
The maximum supported MTU in AWS Top Secret Region is not the same as
AWS commercial. For more information about configuring MTU during isntallation,
see the _Cluster Network Operator configuration object_ section in _Installing
a cluster on AWS with network customizations_
endif::aws-secret[]
====

Reply:
Thanks, Mike! I have removed the nested ifdef and endif and also added the above statement as a note. Please let me know if this looks good in the doc preview! :)